src/backend/looped/session/auto_kick_host.cpp

```diff
@@ -14,7 +14,9 @@ namespace big
 			g_player_service->iterate([](auto& plyr) {
 				if (plyr.second->is_host())
 				{
-					dynamic_cast<player_command*>(command::get(RAGE_JOAAT("breakup")))->call(plyr.second, {});
+					dynamic_cast<player_command*>(command::get(RAGE_JOAAT("bailkick")))->call(plyr.second, {});
```

Comment:
isn't there new way to call commands?

Reply:
There was a PR for that although im not sure if its merged yet.